### PR TITLE
rpc: add more services to getClusterNodes

### PR DESCRIPTION
**Pull request by fanatid**
#### Problem

Not possible to get a TPU forward address for light clients.

#### Summary of Changes

Add `tpu_forwards` and `tpu_forwards_quic` to `getClusterNodes`.

### CODE REVIEW

**Comment by t-nelson** [comment]
mind adding the rest of the missing services from `ContactInfo` while we're here?

* `serve_repair`
* `tpu_vote`
* `tvu`

there are technically a couple more quic ports but those services may never be transitioned

**Comment by fanatid** [comment]
@t-nelson updated

**Comment by t-nelson** [comment]
> there are technically a couple more quic ports but those services may never be transitioned

sry i was unclear. by this comment i meant that we should leave these experimental quic ports out for now. cc/ @behzadnouri

File: rpc-client-api/src/response.rs [hunk 1]
```diff
@@ -294,10 +294,24 @@ pub struct RpcContactInfo {
     pub pubkey: String,
     /// Gossip port
     pub gossip: Option<SocketAddr>,
+    /// Tvu UDP port
+    pub tvu: Option<SocketAddr>,
+    /// Tvu QUIC port
+    pub tvu_quic: Option<SocketAddr>,
```
Comment (behzadnouri) [inline rpc-client-api/src/response.rs]: I don't see much value in exposing this for now, as TVU over QUIC is not rolled out yet.

File: rpc-client-api/src/response.rs [hunk 2]
```diff
@@ -294,10 +294,24 @@ pub struct RpcContactInfo {
...
+    /// Tpu UDP forwards port
+    pub tpu_forwards: Option<SocketAddr>,
+    /// Tpu QUIC forwards port
+    pub tpu_forwards_quic: Option<SocketAddr>,
+    /// Tpu UDP vote port
+    pub tpu_vote: Option<SocketAddr>,
+    /// Server repair UDP port
+    pub serve_repair: Option<SocketAddr>,
+    /// Server repair QUIC port
+    pub serve_repair_quic: Option<SocketAddr>,
```
Comment (behzadnouri) [inline rpc-client-api/src/response.rs]: similarly here.

**Review by behzadnouri — DISMISSED** [review]

**Comment by fanatid** [comment]
@behzadnouri updated

**Review by behzadnouri — APPROVED** [review]

**Review by CriesofCarrots — APPROVED** [review]

**Comment by fanatid** [comment]
Thanks!

**Pull request merged**